### ISSUE TITLE
The "Load/Jump Lazy Template" admin verb causes about 1700 runtimes

### ISSUE THREAD

**Issue by GPeckman** [tgstation/tgstation#86006]
### Client Version:

515.1637

### Issue Summary:

![Screenshot 2024-08-20 183504](https://github.com/user-attachments/assets/32168531-5282-4d8a-b2ec-05f8b236bcac)

### Round ID:

N/A locally tested

### Reproduction Steps:

I loaded the ninja holding facility, didn't try any others.

**Comment by san7890** [comment]
yeah things trying to access the `run_later` var have been an issue for a while in live servers so it's interesting to see that we have a reproducible case now

**Comment by GPeckman** [comment]
> yeah things trying to access the `run_later` var have been an issue for a while in live servers so it's interesting to see that we have a reproducible case now (unless that specific runtime occurred independently of the template load)

No, the runtimes definitely happened when I did the thing. I noticed it immediately because I had the "breakpoint on runtimes" thing checked.
